uber/site_sections/emails.py

```diff
@@ -3,12 +3,15 @@
 
 @all_renderable(c.PEOPLE)
 class Root:
-    def index(self, session, page='1'):
+    def index(self, session, page='1', search_text=''):
         emails = session.query(Email).order_by(Email.when.desc())
+        if search_text:
+            emails = emails.icontains(Email.dest, search_text)
```

Comment:
FYI, this uses a custom function which happens to share a name with some Django stuff. Something I will kick Eli's shins for later.

https://github.com/magfest/ubersystem/blob/master/uber/models.py#L438

In any case, you should also use `.strip()` here for the search_text, as it'll strip away trailing spaces (which mobile keyboards tend to add).